docs/rules/configurable-rules.md

```diff
@@ -22,7 +17,7 @@ rules:
     ...
 ```
 
-A custom rule describes the contents that the linter expects to find in your API definition. During the validation process, the linter goes through your API definition and checks if its contents match the expectations. If something was described in an assertion, but the API definition doesn't correspond to the description, the linter shows you a warning or error message in the log.
+An assertion describes the contents that the linter expects to find in your API definition. During the validation process, the linter goes through your API definition and checks if its contents match the expectations. If something was described in an assertion, but the API definition doesn't correspond to the description, the linter shows you a warning or error message in the log.
```

Comment:
I'm starting to think that the main confusion comes from the `assert/give-the-rule-a-name` syntax. Because THAT is a rule, and it contains assertions and other criteria. I don't want to block this docs PR for a syntax change though, so I'll just try to fix the wording on what we have, and look at whether we could use `rule/my-better-rule` naming instead without too much disruption.

Reply:
I've moved this to #1061 so that we can track it there, and not hold up the docs.